composer.json

```diff
@@ -66,7 +66,9 @@
         "nunomaduro/collision": "^3.0",
         "phpunit/phpunit": "^7.5",
         "squizlabs/php_codesniffer": "^2.9.1",
-        "staudenmeir/dusk-updater": "^1.0"
+        "staudenmeir/dusk-updater": "^1.0",
+        "symplify/coding-standard": "^7.2",
```

Comment:
don't understand. why/where do you need this libraries?
`        "symplify/coding-standard": "^7.2",
        "symplify/easy-coding-standard": "^7.2"`

Reply:
When i was programming those modules i wanted to be able to do some extra checks to the code before uploading it to  github, so, i added those by mistake to the pull request. Good catch, thanks.